### ISSUE TITLE
Why band on Greasyfork?

### ISSUE THREAD

**Issue by wwa4342c1** [adamlui/chatgpt-infinity#84]
Why are you labled as "Banned" on Greasyfork.org? 

https://greasyfork.org/en/users/3967-adamlui

**Comment by adamlui** [comment]
Sup it is because the owner already doesn't like me since on GitHub I exposed issues with his code very often, then one day, someone posted a script with many violations (like over-matching useless sites, against GF rules) so I reported but mods ignored, then I created alt accounts (using my real domain kudoai.com so hiding nothing) to report the rule violator again, this time tagging site owner but unexpectedly, he used it as opportunity to ban me, citing the following:

![image](https://github.com/user-attachments/assets/6845cedd-4be8-4f55-9a59-1cebf633ad4a)

... even though they weren't "puppet" (again, I used emails containing "@kudoai.com" so it was clearly me) and there is no rule against creating accounts to report rule violators

TLDR he really doesn't like me, so much so that even though my reports benefit his site by forcing rule violators to clean up their code, he still found this incident a sufficient way to justify getting rid of me

**Comment by adamlui** [comment]
BTW you can continue using the userscript and updates will auto-sync or use the native browser extensions if you wanted:

https://chrome.chatgptinfinity.com
https://firefox.chatgptinfinity.com
https://edge.chatgptinfinity.com
